src/wombats/sockets/game.clj

```diff
@@ -185,6 +190,68 @@
       (send-message chan-id
                     (m/game-info-message game-state)))))
 
+(defn- build-simulation-game-state
+  [socket-user wombat-id simulator-template-id datomic]
+  (let [simulator-template
+        ((:get-simulator-arena-template-by-id datomic)
+         simulator-template-id)
+
+        user
+        ((:get-entire-user-by-id datomic)
+         (:user/id socket-user))
+
+        wombat
+        ((:get-wombat-by-id datomic)
+         wombat-id)]
+
+    {:arena-config (:simulator-template/arena-template simulator-template)
+     :players {(gen-id) {:player {:player/color "gray"}
```

Comment:
Good call, it's gray because I arbitrarily pick that color pending designs / workflow for the simulator.